.github/workflows/base-release.yaml

```diff
@@ -98,19 +100,19 @@ jobs:
         with:
           fetch-depth: 0
 
-      - uses: sigstore/cosign-installer@v3
```

Comment:
Can you check these downgrades? Were they intentional? Perhaps my version is older than that we have here in this repo